gammapy/spectrum/utils.py

```diff
@@ -199,12 +199,12 @@ def apply_edisp(self):
 
 def integrate_spectrum(func, xmin, xmax, ndecade=100, intervals=False):
     """
-    Integrate 1d function using the log-log trapezoidal rule. If scalar values
-
-    for xmin and xmax are passed an oversampled grid is generated using the
-    ``ndecade`` keyword argument. If xmin and xmax arrays are passed, no
-    oversampling is performed and the integral is computed in the provided
-    grid.
+    Integrate 1d function using the log-log trapezoidal rule.
+    
+    If scalar values for xmin and xmax are passed an oversampled grid is
```

Comment:
This DM PR needs tests and docs and probably a bit of review. Please split out this unrelated cleanup edit in `gammapy/spectrum/utils.py`, Ok to do such small cleanup commits directly in `master`.